modules/raft/src/testFixtures/java/org/apache/ignite/raft/server/counter/IncrementAndGetCommand.java

```diff
@@ -18,29 +18,22 @@
 package org.apache.ignite.raft.server.counter;
 
 import org.apache.ignite.internal.raft.WriteCommand;
+import org.apache.ignite.network.annotations.Transferable;
+import org.apache.ignite.raft.messages.TestMessageGroup;
+import org.apache.ignite.raft.messages.TestRaftMessagesFactory;
 
 /**
  * Increment and get command.
  */
-public class IncrementAndGetCommand implements WriteCommand {
-    /**
-     * The delta.
-     */
-    private final long delta;
-
-    /**
-     * Constructor.
-     *
-     * @param delta The delta.
-     */
-    public IncrementAndGetCommand(long delta) {
-        this.delta = delta;
-    }
-
+//TODO Move to integration test directory when Maven build is not supported anymore.
```

Comment:
There is "todo" without a ticket number.

Reply:
I'll add it, thank you